CONTRIBUTING.rst

```diff
@@ -1,18 +1,30 @@
+.. highlight:: shell
+
 Contributing
 ============
-First clone this repository (optional: and a virtual env).
-Note that we support **Python 3.5+**.
+1. Fork the `cryptoconditions`_ repo on GitHub.
+2. Clone your fork locally and enter into the project::
+
+    $ git clone git@github.com:your_github_handle/cryptoconditions.git
+    $ cd cryptoconditions/
 
-Install in development mode:
+3. Add the ``upstream`` remote::
+    
+    $ git remote add upstream git@github.com:bigchaindb/cryptoconditions.git
 
-.. code-block:: bash
+4. Install in development mode::
 
     $ pip install -e .[dev]
 
-Write your patch ...
+5. Make sure you can run the tests::
+
+    $ pytest -v
 
-Run the tests:
+For the installation step and running the tests you can also use the provided
+``docker-compose.yml`` file::
 
-.. code-block:: bash
+$ docker-compose build
+$ docker-compose run --rm cryptoconditions pytest -v
```

Comment:
For consistency with the other command-line commands, these two lines could be indented as well.